kolibri/plugins/registry.py

```diff
@@ -49,8 +49,8 @@ def initialize():
                 plugin_module = importlib.import_module(app + ".kolibri_plugin")
                 logger.debug("Loaded kolibri plugin: {}".format(app))
                 plugin_classes = []
-                for obj in plugin_module.__dict__.values():
-                    if type(obj) == type and obj is not KolibriPluginBase and issubclass(obj, KolibriPluginBase):
+                for obj in plugin_module.PLUGINS:
```

Comment:
@benjaoming this was necessary, as any plugin classes subclassed from something other than KolibriPluginBase cause the base class to also be imported.

Have changed so that kolibri_plugin.py requires explicit export of the plugin classes defined therein.